test/e2e/framework/pod/wait.go

```diff
@@ -51,7 +51,7 @@ const (
 	podScheduledBeforeTimeout = podListTimeout + (20 * time.Second)
 
 	// podStartTimeout is how long to wait for the pod to be started.
-	podStartTimeout = 5 * time.Minute
+	podStartTimeout = 60 * time.Minute
```

Comment:
This is going to substantially increase the amount of time it takes e2e tests to fail across several test suites. Is this 12x bump (one hour!) needed specifically for the pods in the Windows device plugin tests introduced in this PR? If so, can you please scope this Timeout for the specific tests in the PR following a pattern like https://github.com/kubernetes/kubernetes/blob/b8ecff28d30344fc22b6a2275f85b2340f6bf2a6/test/e2e/storage/testsuites/volumelimits.go#L190